mindmeld/active_learning/plot_manager.py

```diff
@@ -494,8 +544,8 @@ def plot_stacked_bar(
             plot_domain (bool): Whether to generate plots at the domain level.
             plot_intents (bool): Whether to generate plots at the intent level.
         """
-        for strategy in self.queries_data:
-            epoch_dict = self.queries_data[strategy][str(epoch)]
+        for strategy in self.strategies[0]:
```

Comment:
When we're accessing the first index value here, is it guaranteed to exist?

Reply:
Yup, added checks to ensure that!